src/App/routes/patterns.js

```diff
@@ -29,6 +29,14 @@ module.exports = [
                 icon: "account_circle",
                 outlined: true,
                 statusBadges: ["new"]
+            },
+            {
+                title: "Filters",
+                path: "/patterns/filters",
+                componentPath: "content/Filters",
+                icon: "filter_alt",
+                outlined: true,
+                statusBadges: ["new"]
```

Comment:
Hmmm..Should this be on this branch? 🤔

Reply:
haha! I DONT KNOW 😂

Reply:
This should be removed from this branch aswell